html_css/v2/main-course/browser-compatibility.md

```diff
@@ -0,0 +1,70 @@
+
+### What is Browser Compatibility
+Today, it is impossible to imagine the Web without a use of the browsers. We have witnessed a shift from standalone applications to HTML5 and Progressive Web Apps that allow applications to fully function within a browser. For example, Microsoft Word and Excel for the longest time could only be executed via a standalone application. Now, you can utilize those applications through any browsers without a need to install any files. 
+
+As companies compete for the market share, different browsers are using different engines to display information on the web page. For example, Chrome and chromium utilize Blink, while Safari uses its proprietary WebKit.
+
+Because of these differences, your applications may behave differently in the browser. Due to Chrome dominance, vast majority of applications are designed to work smooth with chromium and providing as good of performance in other browsers is secondary. 
+
+For your web development projects to have a broader reach, you must make sure that you're testing your web applications against the browsers which are most likely to be used by the users. Internet Explorer is the most common browser in large companies. While Chrome, Safari, Firefox and other chromium-based browsers (Microsoft Edge, Brave, etc.) are more common among regular users. For Chromium browsers, if it works in Chrome, it should work in other related browsers as well. 
+
+### Browser Releases and New CSS Features
+W3C [World Wide Web Consortium]( https://www.w3.org/) is the authority behind developing web standards to maximize accessibility and consistency of the web experience. W3C is also the authority developing new features in the CSS. This is a closely collaborative approach with the Web as a community as well the companies developing the web browsers. 
+
+When the web browsers such as Nexus and Netscape were released, there was no such organization as W3C to help create more compatibility. Your application could look and function differently in each browser. Even worse, your application could've been completely unusable. Web developers had to make specific adjustments for each browser, and not every developer had enough resources to make that work for everyone.
+
+Today, as the standards around the web evolve and change, and web developers begin implementing new features in their codebase, it is imperative for the browsers to provide support for those new features. If the user experience is impacted by the lack of support in the browsers, the users might find their way to the competitor.
+
+### When is It Safe to Use New Features
+As exciting as it is to implement new features, there is a risk to rushing. It would not be a positive experience for your users to find that, for example, your application used to work well in Firefox, but due to the changes in the codebase it is now unusable in Firefox but works well in Safari. Thankfully, there is a tool that can help you prevent this situation. 
```

Comment:
```suggestion
As exciting as it is to implement new features, there is a risk of rushing. It would not be a positive experience for your users to find that, for example, your application used to work well in Firefox, but due to the changes in the codebase it is now unusable in Firefox but works well in Safari. Thankfully, there is a tool that can help you prevent this situation. 
```